### PR TITLE
adding the critical incident management policy

### PR DESCRIPTION
**Pull request by denispop9**
- [x] #

 Description

I added the critical incident management policy.

## What

A new policy will appear on the website.

## Why

It is being made because it is one of my tasks.

## How

Using Github, html and sublime.

## Verification Steps

What steps were taken to verify the change is working correctly

# Checklist:

 ] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by denispop9** [comment]
@laurafitzgerald can you have a look at this for me?

**Comment by dominikkawka** [comment]
Could you put up a screenshot of the page?

**Comment by laurafitzgerald** [comment]
@denispop9 some formatting issues in your description, just checking the changes locally now.

File: pages/about/policies/criticalincidentmanagement.html [hunk 1]
```diff
@@ -1 +1,1332 @@
-
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```
Comment (laurafitzgerald) [inline pages/about/policies/criticalincidentmanagement.html]: @denispop9 I'm assuming the content is fine but we don't need all the styling that you've included, please review the changes in the original pr that is linked in the trello card which will show how the other files were added.

File: pages/about/policies/criticalincidentmanagement.html [hunk 2]
```diff
@@ -1 +1,873 @@
...
+<p>1800 411 057</p>
+</td>
+</tr>
+</tbody>
+</table>
+<p><br /><br /></p>
+<div>
+<p><br /><br /></p>
+</div>
+<p>The Rubik's Cube is a three-dimensional twisty puzzle. Learn the easiest <a href="https://ruwix.com/the-rubiks-cube/how-to-solve-the-rubiks-cube-beginners-method/">Rubik's Cube solution here</a>.</p>ss
```
[the remark this answers is not in the document]
Comment (laurafitzgerald) [reply pages/about/policies/criticalincidentmanagement.html]: @denispop9 can leave the comment unresolved until the change is made and pushed. github resolves them automatically when the requested change is fixed.

File: pages/about/policies/criticalincidentmanagement.html [hunk 3]
```diff
@@ -1 +1,873 @@
-
+<p>Responding to Critical Incidents Resource Materials for Schools (File Format PDF 170KB)</p>
+<p><br /><br /></p>
```
[the remark this answers is not in the document]
Comment (laurafitzgerald) [reply pages/about/policies/criticalincidentmanagement.html]: @denispop9 same here

**Comment by laurafitzgerald** [comment]
@denispop9 thanks for the new changes. This is starting to come together now. There are still a few small formatting issues, for example in between some of the paragraphs there is a lot of space between. There is also a small section of text on the policies.html page which needs to be added. You can view the original pr from me which is linked in the trello card to find out how to add this.

**Pull request merged**